src/Symfony/Component/Mime/Encoder/ContentEncoderInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime\Encoder;
+
+/**
+ * Interface for all Transfer Encoding schemes.
+ *
+ * @author Chris Corbyn
+ */
+interface ContentEncoderInterface extends EncoderInterface
+{
+    /**
+     * Encodes $os to a Generator.
+     *
+     * @param resource $os
```

Comment:
what does `$os` stands for? octet stream?

Reply:
output stream actually, renamed to just `$stream`